cli/tests/pcluster/api/controllers/test_cluster_operations_controller.py

```diff
@@ -1033,29 +1034,411 @@ def test_aws_api_errors(self, client, mocker):
 
 
 class TestUpdateCluster:
```

Comment:
Shall we add a test for force update?

Reply:
The successful test does check that the `Cluster.update` is called with the `forceUpdates` that arrives to the controller. The behavior with the `force` is tested in the tests for the `Cluster` class, but we did not change that functionality in this patch.